ratis-metrics-dropwizard3/pom.xml

```diff
@@ -25,7 +25,7 @@
   <name>Apache Ratis Metrics Dropwizard 3 Implementation</name>
 
   <properties>
-    <dropwizard3.version>3.2.5</dropwizard3.version>
+    <dropwizard3.version>4.2.7</dropwizard3.version>
```

Comment:
Since dropwizard 4 is incompatible to dropwizard 3,  `ratis-metrics-dropwizard3` must stay in dropwizard 3.  For dropwizard 4, please use `ratis-metrics-default` instead.  It will use the shaded dropwizard (`org.apache.ratis.thirdparty.com.codahale.metrics.*`) which is dropwizard 4.2.9; see https://github.com/apache/ratis-thirdparty/blob/52f2fe29ff91ad94d5d9a690cba2d4d4ec3ce024/pom.xml#L77